GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift

```diff
@@ -19,13 +19,17 @@ struct DogBreedsListView: View {
                 Text("min life: \(dogBreed.minLife), max life: \(dogBreed.maxLife)")
             }
         }
+        .onAppear(perform: {
+            viewModel.initialFetch()
+        })
+        .edgesIgnoringSafeArea(.bottom)
     }
 }
 
 struct DogBreedsListView_Previews: PreviewProvider {
     static var previews: some View {
-        DogBreedsListView(
-            viewModel: DogBreedsListViewModel(dogBreedsService: DogBreedsService())
+        MainView(
```

Comment:
Why did you change it to `MainView`? it was correct

Reply:
I took it from reddit, i thought it was neeeded.

Reply:
Yes, RedditView may have it incorrectly then, we can fix it afterwards. Not part of this pr 👍